fortran/test/fetch_content_integration/test_micm_api.F90

```diff
@@ -172,27 +173,27 @@ subroutine test_multiple_grid_cells(micm, NUM_GRID_CELLS, time_step, test_accura
 
     type(micm_t), pointer, intent(inout) :: micm
     integer,               intent(in)    :: NUM_GRID_CELLS
-    real(c_double),        intent(in)    :: time_step
+    real(real64),        intent(in)    :: time_step
     real,                  intent(in)    :: test_accuracy
 
     integer, parameter        :: NUM_SPECIES = 6
     integer, parameter        :: NUM_USER_DEFINED_REACTION_RATES = 2
-    real(c_double), target    :: temperature(NUM_GRID_CELLS)
-    real(c_double), target    :: pressure(NUM_GRID_CELLS)
-    real(c_double), target    :: air_density(NUM_GRID_CELLS)
-    real(c_double), target    :: concentrations(NUM_GRID_CELLS * NUM_SPECIES)
-    real(c_double), target    :: initial_concentrations(NUM_GRID_CELLS * NUM_SPECIES)
-    real(c_double), target    :: user_defined_reaction_rates(NUM_GRID_CELLS * NUM_USER_DEFINED_REACTION_RATES)
+    real(real64), target    :: temperature(NUM_GRID_CELLS)
+    real(real64), target    :: pressure(NUM_GRID_CELLS)
+    real(real64), target    :: air_density(NUM_GRID_CELLS)
+    real(real64), target    :: concentrations(NUM_GRID_CELLS * NUM_SPECIES)
+    real(real64), target    :: initial_concentrations(NUM_GRID_CELLS * NUM_SPECIES)
+    real(real64), target    :: user_defined_reaction_rates(NUM_GRID_CELLS * NUM_USER_DEFINED_REACTION_RATES)
     type(string_t)            :: solver_state
     type(solver_stats_t)      :: solver_stats
     integer(c_int)            :: solver_type
```

Comment:
Would it be just `integer`? 
```Fortran
integer :: solver_type
```
It's a minor request, but could we use consistent spacing for the listed variables to improve visual clarity?

Reply:
fixed, and reformatted